resources/metadata/malwareExecuteScan.yaml

```diff
@@ -44,3 +44,12 @@ spec:
         - STAGES
         - STEPS
         mandatory: true
+      - name: timeout
+        type: string
+        description: "timeout for http layer in seconds"
+        scope:
+        - PARAMETERS
+        - STAGES
+        - STEPS
+        mandatory: false
+        default: 600
```

Comment:
May be 600 sec too high?

Reply:
... suggest another more appropriate value.

Reply:
300s ?

Reply:
Is it not required to update this new parameter in the docu?

Reply:
To my knowledge the docu is created based on the metadata. To my understanding no furhter docu update required.